core/App/screens/ListContacts.tsx

```diff
@@ -1,34 +1,49 @@
 import { ConnectionRecord, ConnectionType } from '@aries-framework/core'
 import { useConnections } from '@aries-framework/react-hooks'
+import { StackNavigationProp } from '@react-navigation/stack'
 import React from 'react'
-import { FlatList } from 'react-native'
+import { FlatList, StyleSheet, View } from 'react-native'
 
 import ContactListItem from '../components/listItems/ContactListItem'
 import EmptyList from '../components/misc/EmptyList'
 import { useStore } from '../contexts/store'
 import { useTheme } from '../contexts/theme'
+import { ContactStackParams, Screens } from '../types/navigators'
 
 interface ListContactsProps {
-  navigation: any
+  navigation: StackNavigationProp<ContactStackParams, Screens.Contacts>
 }
 
 const ListContacts: React.FC<ListContactsProps> = ({ navigation }) => {
+  const { ColorPallet } = useTheme()
+  const style = StyleSheet.create({
+    list: {
+      backgroundColor: ColorPallet.brand.secondaryBackground,
+    },
```

Comment:
The background of the list should not be the same as the background of the items, right? Currently, both use `ColorPallet.brand.secondaryBackground`

Reply:
The list of credentials looks different because there are cards.  
We made a list of contacts to look like the list of credential attributes now and list of request attributes/predicates before switching to cards